### DISCUSSION TITLE
landing page Games request - please change the descriptions of bets

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#20] [Bugs and Reports]
please. Change the descriptions of "active BETS" for LAST 24 hours games or 24h historical??... or just make 2 rows of numbers like ACTIVE BETS and 24H BETS

<img width="983" alt="Snímek obrazovky 2024-09-30 v 11 19 51" src="https://github.com/user-attachments/assets/82195845-7943-42df-88cd-9db7448687bc">

**Comment by bf-monster** [comment]
Sure, we will discuss this. thank you
